blueprints/acceptance-test/index.js

```diff
@@ -1,3 +1,13 @@
+var existsSync = require('exists-sync');
+
 module.exports = {
-  description: 'Generates an acceptance test for a feature.'
+  description: 'Generates an acceptance test for a feature.',
+
+  locals: function() {
+    var destroyAppExists = existsSync(this.project.root + '/tests/helpers/destroy-app.js');
```

Comment:
Should probably use `path.join` here. Node *might* do the right thing on Windows, but I'm never certain...